SKILL.tsv

```diff
@@ -791,7 +791,7 @@ SKILL_20150317_000790	$Attack an enemy with magic from a far distance.
 SKILL_20150317_000791	$Attack an enemy from a short distance(Warrior's Two-handed Sword only)
 SKILL_20150317_000792	$Attack an enemy magic from a far distance.(Two-handed Staff)
 SKILL_20150317_000793	$Dagger Pierce
-SKILL_20150317_000794	$Attack an enemy from a short distance. Knockbacks the target when it is struck a certain number of times.
+SKILL_20150317_000794	$Attack an enemy from a short distance. Knock back the target when it is struck a certain number of times.
```

Comment:
These translations are the only place I have actually seen "knockbacks" when used as a verb - "Knockbacks the enemy". Looking on GW2 Wiki, Secret World, Perfect World wiki they either describe the act of being knocked back as being pushed or writing it as "Knock the enemy back". If it was just "This applies a knockback" or "Archer has too many knockbacks" (used as a noun) I would agree, but "knockbacks" and "knockbacked" as verbs sound weird.